src/dmd/backend/ee.d

```diff
@@ -14,8 +14,8 @@ module dmd.backend.ee;
  * Code to handle debugger expression evaluation
  */
 
-version (SPP) {} else
-{
+version (SPP) {}
+else:
```

Comment:
Unrelated.  And I'm not sure what the convention is.

Reply:
IDK either, it covers the whole file though so I removed it.